### ISSUE TITLE
Future state of the app

### ISSUE THREAD

**Issue by MF-Debug** [CrazyBoyFeng/AccSettings#13]
What is the current status of this app? Is the development stopped? It hasn't received any update for a long time.

**Comment by Releece** [comment]
Yes we would love a little update! ACCD also didn't release any updates for a long time, so there was less need for this app to get any development, but VR-25 is on fire again since this summer. Which makes me really happy:) But I really hope the GUI can get some love as well.

Irregardless, thanks for giving us this app and all the time you put in it already. And if you're not in the mood / don't have time anymore, maybe someone else can pick up the development? We need to save all those sad pieces of Lithium around the world from heavy abuse: 120Watt speed charging everyday??? What is happening to the world :O Those cruel bastards... Whats next, chargers are going to be heavier than the frickin phones? 

Sadly, CLI-only will not reach nearly as much people as a GUI can

**Comment by CrazyBoyFeng** [comment]
Sorry I didn't continue development this year for personal reasons. Development will gradually resume next, but please understand that it won't be soon.

**Comment by Releece** [comment]
Take your time!
Happy to know you're still planning further development, whenever that time comes.
Thanks for the status update

**Comment by thejohnha** [comment]
Hi CBF, just kindly checking in to see if you might have any time in 2025 for development on your awesome front-end for acc. Thanks so much :-)
